docs/conf/prague/2018/job-fair.rst

```diff
@@ -0,0 +1,11 @@
+:template: {{year}}/generic.html
+
+Job Fair
+========
+
+.. include:: /include/conf/events/job-fair.rst
```

Comment:
btw there's a typo in the job fair snippet... see if you can spot it ;) (hint: it's a food joke)